### DISCUSSION TITLE
pre-built Windows binaries?

### DISCUSSION THREAD

**Discussion by jchang6** [opcm/pcm#482] [Q&A]
I cannot find this on the appveyor site? https://ci.appveyor.com/project/opcm/pcm/history
any windows install directions would be appreciated
if any has a visual studio proj file, that would be appreciated as well
thanks

**Comment by ogbrugge** [comment]
Visual Studio project files are created once you run cmake. 

Please read https://github.com/intel/pcm/blob/master/doc/WINDOWS_HOWTO.md
